packages/renumerate/lib/core.ts

```diff
@@ -378,7 +385,10 @@ export class Renumerate {
 	 */
 	private addListener() {
 		window.addEventListener("message", (event) => {
-			const allowedOrigins = ["https://renumerate.com"];
+			const isLocal = this.getIsLocal();
+			const allowedOrigins = isLocal
+				? ["http://localhost:3000", "http://localhost:4321"]
+				: ["https://renumerate.com"];
```

Comment:
feels like this needs to be retention. and subs. not the root one anymore right?